kola/tests/coretest/core.go

```diff
@@ -275,6 +296,49 @@ func servicesActive(units []string) error {
 	return nil
 }
 
+func TestServicesDisabledRHCOS() error {
+	inactiveErr := servicesInactive(offServices)
+	if inactiveErr != nil {
+		return inactiveErr
+	}
+
+	disabledErr := servicesDisabled(offServices)
+	if disabledErr != nil {
+		return disabledErr
+	}
+	return nil
+}
+
+func servicesInactive(units []string) error {
+	for _, unit := range units {
+		c := exec.Command("systemctl", "is-active", unit)
+		err := c.Run()
+		if err == nil {
+			return fmt.Errorf("Service Incorrectly Active: %q", unit)
+		}
+	}
+	return nil
+}
+
+func servicesDisabled(units []string) error {
+	for _, unit := range units {
+		c := exec.Command("systemctl", "is-enabled", unit)
+		var out bytes.Buffer
+		c.Stdout = &out
```

Comment:
Any reason to do this rather than `out, err := c.Output()`?

Reply:
Just learning my way around golang...the example from the docs had this way, but I like the one-line approach much better.